features/deduper.js

```diff
@@ -0,0 +1,61 @@
+const deduper = {
+  monitoredChannels: [
+    "102860784329052160", // #general
+    "103696749012467712", // #need-help-0
+    "565213527673929729" // #need-help-1
+  ],
+
+  dupeExpireTime: 2 * 60 * 1000, // 2 minutes to reject the duplicate as cross-post (ms)
+  cleanerInterval: 10 * 60 * 1000, // Frees up the memory from 10 to 10 minutes (ms)
+
+  lastMessageOfMember: {},
+
+  deleteMsgAndWarnUser: (msg, lastMsg) => {
+    msg.delete();
```

Comment:
Feeling like we should delete the last message, so if they cross post only the latest survives. feel like that'd be a little more subtle, so the folks who ignore their DMs and quickly spam don't get wise to what's happening.

```suggestion
    lastMsg.delete();
```

Reply:
Wouldn't it be more confusing to automatically delete the last message without the user knowing (since he is not looking at the other channel by the time it is deleted)?

Reply:
Exactly. It's less visible, which in this case I think is a benefit. They're notified by the DM, not by the immediacy of seeing their message disappear.

Reply:
But isn't it kinda unexpected? One user may have already answered the question then the message suddenly disappears because the poster sent it on another channel, idk 🤔

Reply:
If a handful of minutes that feels like an uncommon case 🤷‍♂ I'm okay with unexpected things happening when people behave badly

Reply:
Done! Please check the bot message, I am not sure it is clear enough or I wrote something wrong.